pom.xml

```diff
@@ -1120,6 +1120,10 @@
 						<showDeprecation>true</showDeprecation>
 						<showWarnings>true</showWarnings>
 						<verbose>false</verbose>
+						<fork>true</fork>
+						<useIncrementalCompilation>true</useIncrementalCompilation>
+
+
```

Comment:
These two empty lines could be removed!

Reply:
`useIncrementalCompilation` is already `true`. IMO there is no need to be explicit.